lib/asciidoctor/timings.rb

```diff
@@ -36,5 +36,16 @@ def print_report to = $stdout, subject = nil
       to.puts %(  Time to convert document: #{'%05.5f' % convert.to_f})
       to.puts %(  Total time (read, parse and convert): #{'%05.5f' % read_parse_convert.to_f})
     end
+
+    if ::Process.respond_to? :clock_gettime
+      CLOCK_ID = ::Process::CLOCK_MONOTONIC
```

Comment:
@mojavelinux `clock_gettime` is defined in an Opal environment but not `::Process::CLOCK_MONOTONIC`:

```
CLOCK_MONOTONIC: uninitialized constant Process::CLOCK_MONOTONIC
```

https://travis-ci.org/asciidoctor/asciidoctor.js/jobs/345568804

Reply:
Dang. What are the chances? That's okay, we can just check for both.

Reply:
JavaScript can use Time.now with good accuracy, so this is just a Ruby thing.

Reply:
I see a reference to CLOCK_MONOTONIC (which is mapped to performance.now()). 

https://github.com/opal/opal/blob/478f5f15e41ef8ba398d995846532b17aed82be1/opal/corelib/process.rb#L30

Is that not in a release yet?

Reply:
Indeed but the constant was removed: https://github.com/opal/opal/commit/319f9c07ea0111e2f8488903cfc0f585d28c2847

> Is that not in a release yet?

I need to check if it's available in Asciidoctor.js.

> I see a reference to CLOCK_MONOTONIC (which is mapped to performance.now()).

I don't know if it's safe to use this implementation because `performance.now` and `process.hrtime` may not be available in a Nashorn environment.

Reply:
Good point. We can always put an Opal guard on it if necessary.